ruma-api-macros/src/api/request/outgoing.rs

```diff
@@ -131,23 +131,31 @@ impl Request {
             .collect();
 
         for auth in &metadata.authentication {
-            if auth.value == "AccessToken" {
-                let attrs = &auth.attrs;
-                header_kvs.extend(quote! {
+            let attrs = &auth.attrs;
+            let auth_value = &auth.value.to_string();
+            header_kvs.extend(quote! {{
```

Comment:
Using `{{` here to have shadowing of `access_token` in it's own scope as it makes problems otherwise